bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelectFromMultiSelect.php

```diff
@@ -65,18 +68,19 @@ public function addCondition(AbstractFilterDefinitionType $filterDefinition, Pro
         $currentFilter[$field] = $value;
 
         if (!empty($value)) {
-            if ($filterDefinition->getUseAndCondition()) {
-                foreach ($value as $entry) {
-                    $productList->addCondition(['term' => ['attributes.' . $field => $entry]], $field);
-                }
-            } else {
-                $boolArray = [];
-                foreach ($value as $entry) {
-                    $boolArray[] = ['term' => ['attributes.' . $field => $entry]];
-                }
+            if (method_exists($filterDefinition, 'getUseAndCondition'))
```

Comment:
if has no {} ....

besides that, we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelectFromMultiSelect`, other wise throw `InvalidConfigException`